Cabal/src/Distribution/PackageDescription/Check/Monad.hs

```diff
@@ -48,7 +48,7 @@ import Distribution.CabalSpecVersion (CabalSpecVersion)
 import Distribution.Package (packageName)
 import Distribution.PackageDescription.Check.Warning
 import Distribution.Simple.BuildToolDepends (desugarBuildToolSimple)
-import Distribution.Simple.Glob (Glob, GlobResult)
+import Distribution.Simple.Glob
```

Comment:
Is thre a good reason not to have explicit imports?

Reply:
Sorry, I put them back to what they were before.